devices.js

```diff
@@ -4691,6 +4691,23 @@ const devices = [
         toZigbee: [],
     },
 
+    // Centralite
+    {
+        zigbeeModel: ['3323-G'],
+        model: '3323-G',
+        vendor: 'Centralite',
+        description: 'Micro-Door sensor',
+        supports: 'contact and temperature',
+        fromZigbee: [fz.centralite_3323G_contact, fz.temperature],
```

Comment:
Please try with `fz.ias_contact_alarm_1` instead of `fz.centralite_3323G_contact`